### ISSUE TITLE
Suggest use of a bypass block (skip to main content link)

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#849]
**Describe the bug**
Suggest a bypass block on this page, because…

1. Interactive UI owned by OpenFL adds a minimum half dozen tab stops in default sidebar (assuming access to bottom collapsible section is fixed)
2. “Ethical ads” ad plugin in sidebar adds 4 tab stops to overall focus order
3. Use of previous/next buttons in a given page of content will reload entire page, returning user to top of page and making them tab a lot to get to newly presented main content

Therefore, to enable a keyboard user to reach the main content within a couple of keypresses instead of over 10, suggest a 'skip to main content' link in the header of the page, as the first interactive element on the page. 

The skip navigation link can be coded so that it is not visible unless it has keyboard focus, so as not to change the default appearance / layout of the page. Doing this will enable the keyboard user to press TAB to get to the skip link and ENTER to move focus to the first interactive element in the main content.

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.

**Comment by fstrr** [comment]
This is an issue with the [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).

**Comment by theakshaypant** [comment]
Closing. As mentioned by @fstrr, this is an issue with [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).
